.travis.yml

```diff
@@ -6,8 +6,8 @@ env:
   - TRAVIS_JDK=11
 
 before_install: curl -Ls https://git.io/jabba | bash && . ~/.jabba/jabba.sh
-install: jabba install "adopt@~1.$TRAVIS_JDK.0-0" && jabba use "$_" && java -Xmx32m -version
-script: scripts/test-sbt
+#install: jabba install "adopt@~1.$TRAVIS_JDK.0-0" && jabba use "$_" && java -Xmx32m -version
+#script: scripts/test-sbt
```

Comment:
@mkurz, the tests are broken. I'm disabling it for now.

Reply:
:+1:

Reply:
hmm, I will revert it. Seems to be something on my machine.